lib/JSON/Validator/Formats.pm

```diff
@@ -31,8 +31,10 @@ sub check_date {
 }
 
 sub check_date_time {
-  my @dt = $_[0] =~ m!^(\d{4})-(\d\d)-(\d\d)[T ](\d\d):(\d\d):(\d\d(?:\.\d+)?)(?:Z|([+-])(\d+):(\d+))?$!io;
+  my @dt = $_[0] =~ m!^(\d{4})-(\d\d)-(\d\d)[T ](\d\d):(\d\d):(\d\d(?:\.\d+)?)(?:Z|([+-])(\d\d):(\d\d))?$!io;
```

Comment:
I'd prefer \d{2} for readability

Reply:
\d\d was there in the first place and, tbh, I prefer \d\d for readability and only start \d{n} when n>2.